Changes

```diff
@@ -171,9 +175,11 @@ OCaml 4.05.0 (TBD):
   ListLabels, MoreLabels, StringLabels so they are compatible with
   non-labeled counterparts. Also add missing @@ocaml.deprecated attributes
   in StringLabels and BytesLabels.
-  (Roma Sokolov)
+  (Roma Sokolov, review by Gabriel Scherer, Jacques Garrigue,
+   Gabriel Radanne, Alain Frisch)
```

Comment:
I'm not sure if there is a policy, but note that some multi-line credits entries align under the `(` instead of after the `(` as here.